src/platform/forms-system/test/js/review/ObjectField.unit.spec.jsx

```diff
@@ -368,15 +521,13 @@ describe('Schemaform review: ObjectField', () => {
       />,
     );
 
-    expect(
-      tree.subTree('.form-review-panel-page-header-row').subTree('.edit-btn')
-        .props['aria-label'],
-    ).to.equal('Custom label');
+    expect(tree.getByLabelText('Custom label')).to.exist;
```

Comment:
This seems like it is no longer checking `aria-label`. Could we do something like:

```js
expect(tree.querySelector('[aria-label="Custom label"]')).to.exist
```